README.md

```diff
@@ -73,6 +73,9 @@ Below you will find descriptions of all rules available in **AEM Rules for Sonar
 - **AEM-15** Usage of ``synchronized`` keyword should be avoided if possible.
   - Usage of ``synchronized`` keyword should be avoided if possible. Check if using ``synchronized`` can be replaced with more sophisticated solution.
 
+- **AEM-16** Optional is defined as DefaultInjectionStrategy
```

Comment:
The other rules in this file seem to have class names pre-formatted as code when mentioned. This one sticks out a little.